src/lib.rs

```diff
@@ -4686,6 +4686,42 @@ mod tests {
       "@font-face { font-family: 'revert-layer'; }",
       "@font-face{font-family:\"revert-layer\"}",
     );
+
+    prefix_test(
+      r#"
+      .foo {
+        font-family: Helvetica, system-ui, sans-serif;
+      }
+    "#,
+      indoc! {r#"
+      .foo {
+        font-family: Helvetica, system-ui, AppleSystem, BlinkMacSystemFont, Segoe UI, Roboto, Noto Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
```

Comment:
I guess system-ui should be first?

Reply:
I put it second on purpose, this makes sure that the fonts are added at the right place and don't remove the existing. Also, not everybody has Helvetica so it could make sense to want Helvetica but fallback to whatever system font the person has.

Reply:
Ah, sorry, I misread the input. Thanks!